backend/sftp/sftp.go

```diff
@@ -164,6 +164,16 @@ Home directory can be found in a shared folder called "home"
 			Default:  false,
 			Help:     "Set to skip any symlinks and any other non regular files.",
 			Advanced: true,
+		}, {
+			Name:     "subsystem",
+			Default:  "sftp",
+			Help:     "Specifies the SSH2 subsystem on the remote host.",
+			Advanced: true,
+		}, {
+			Name:     "sftp_server",
```

Comment:
Can we call this something other than `sftp_server` otherwise the option will be called `--sftp-sftp-server`! Maybe `server` or `server_command`?